integration-tests/src/tests/client/features/limit_contract_functions_number.rs

```diff
@@ -70,8 +70,8 @@ fn verify_contract_limits_upgrade(
 
 // Check that we can't call a contract exceeding functions number limit after upgrade.
 // Disabled in nightly due to https://github.com/near/nearcore/issues/8590
-#[cfg(all(not(feature = "nightly"), not(feature = "statelessnet_protocol")))]
 #[test]
+#[ignore]
```

Comment:
I commented on #8590 mentioning that I'm disabling this test. It can be fixed in due time.